static/app/views/starfish/modules/APIModule/endpointTable.tsx

```diff
@@ -44,9 +49,9 @@ const COLUMN_ORDER = [
     width: 600,
   },
   {
-    key: 'tpm',
-    name: 'tpm',
-    width: COL_WIDTH_UNDEFINED,
+    key: 'throughput',
```

Comment:
not sure if we should add it to the default column list/API view! adding it to the `HTTP_SPAN_COLUMN_ORDER` in static/app/views/starfish/views/webServiceView/endpointDetails/index.tsx should suffice

Reply:
I think it provides some useful context so it doesn't hurt to add it here as well, I'll ask the rest of the team what they think about leaving it here or removing it